resources/views/layouts/app.blade.php

```diff
@@ -0,0 +1,39 @@
+<!DOCTYPE html>
+<html lang="ja">
+    <head>
+        <meta charset="utf-8">
+        <title>MessageBoard</title>
+        <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
+        <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.2.1/css/bootstrap.min.css">
+        <link rel=”stylesheet”  href="app.css”>
```

Comment:
```suggestion
        <link rel="stylesheet"  href="app.css">
```

おそらく　`` ” `` で囲んでるからで、 `` " `` で囲んでみたらどうですかね？

Reply:
@rui1031 Consoleで確認してみたら、ここが怪しそうなので、試してみてください！
<img width="1151" alt="スクリーンショット 2020-08-15 22 34 28" src="https://user-images.githubusercontent.com/12184184/90313422-7d88a280-df47-11ea-9c70-374ebbfce7ae.png">